<!-- Launch file for visualizing OSM data

     arg: map_url = URL for map data

     $Id$
  -->

<launch>

  <arg name="map_url" default="package://osm_cartography/tests/prc.osm" />

  <!-- start map server -->
  <node pkg="osm_cartography" type="osm_server.py" name="osm_server" />

  <!-- start map visualization -->
  <node pkg="osm_cartography" type="viz_osm.py" name="viz_osm" >
    <param name="map_url" value="$(arg map_url)" />
  </node>

  <!-- start dynamic reconfigure GUI -->
  <node pkg="dynamic_reconfigure" type="reconfigure_gui" name="reconfigure_gui"
        args="viz_osm" />

  <!-- start rviz (not an explicit dependency in this package)
  <node pkg="rviz" type="rviz" name="rviz" />
  -->

</launch>
